proposal.tex

```diff
@@ -1,2 +1,30 @@
 \section{Community Proposal Process}\label{sec:process}
-% Melissa
\ No newline at end of file
+% Melissa
+
+\textbf{Timeline --}
+As the distribution of subthreshold alerts is planned to begin in year 2 of the LSST, the first community proposals will be solicited during the first half of year 1.
+The first deadline will be timed to match the midpoint of year 1 (i.e., the observational cutoff date for Data Release 1).
+Thereafter, proposals would be solicited on a yearly basis.
+
+\textbf{Proposals --}
+The community will be asked to provide proposals for subthreshold alerts that includes three components:
+
+\begin{enumerate}
+\item The anticipated science impact, with a motivation regarding why the science goals require alerts within 60 seconds and cannot be achieved using the Prompt Products Database, which is updated within 24 hours.
+\item A detailed description of how the subthreshold alerts would be identified, e.g., lists of coordinates with inclusion radii, sky bounding box, or other difference-image source detection or measurement parameters.
+\item A consideration of the technical considerations that speaks to the feasibility of the proposed subthreshold alerts.
+\end{enumerate}
+
+In keeping with best practices for research inclusion, the proposals will be anonymized (e.g., using the NOIRLab TAC system or similar).
+
+As alerts are the only fully public LSST data product, the proposals will also be public.
+
+A more detailed call for proposals and a template will be provided at the start of LSST Operations.
+
+\textbf{Evaluation --} A technical review for proposal feasibility will be done first, by a small committee within Rubin Data Production team that is headed by the lead of Alert Production.
+If, after this technical review, the requests for subthreshold alerts are oversubscribed, the Rubin Observatory Director will assemble a five- to six-person scientific review committee.
```

Comment:
> five- to six-person
I'd just say 'small' or 'focused' review committee at this stage - no need to commit ourselves this far in advance. The key point is that the committee should contain the necessecary expertise to  make a decision.

Reply:
makes sense; i went with "small"